index.js

```diff
@@ -2,14 +2,14 @@ const { dialog, globalShortcut } = require('electron');
 const getShortcutFromConfig = require('./modules/getShortcutFromConfig');
 
 function registerShortcut (configKey, action) {
-  return (app) => {
+  return (app, ...args) => {
     const { plugins, config } = app;
     function register (accelerator) {
       if (!accelerator) return;
       globalShortcut.unregister(accelerator);
       const registered = globalShortcut.register(
         accelerator,
-        () => action(app)
+        () => action.apply(this, [app, ...args])
```

Comment:
Yup I agree that the user shouldn't have to destructure those arguments. I think spreading `args` in the function call instead of just passing it might also do the trick though?

So:

```js
() => action(app, ...args)
```

instead of 

```js
() => action(app, args)
```

Unless Node doesn't support this yet! I haven't tried it out myself